packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -148,40 +187,71 @@ export const TranslationSidebar = ({
-                        {t('translate:note_description', {
-                          timestamp: translatorNote?.timestamp
-                            ? new Date(
-                                translatorNote?.timestamp
-                              ).toLocaleString()
-                            : '',
-                          authorName: translatorNote?.authorName ?? '',
-                        })}
-                      </span>
-                    )}
-                    {canEditNote ? (
-                      <RichTextInput
-                        key={word.id}
-                        name="noteContent"
-                        value={noteContent}
-                        onBlur={async (e) => {
-                          saveNote(e.target.value);
-                          saveNote.flush();
-                        }}
-                        onChange={async (e) => {
-                          saveNote(e.target.value);
-                        }}
-                      />
-                    ) : (
-                      <RichText content={noteContent} />
-                    )}
+                    <div className="flex flex-col gap-2">
+                      <h2 className="font-bold">
+                        {t('translate:translator_notes')}
+                      </h2>
+                      {translatorNote?.authorName && (
+                        <span className="italic">
+                          {t('translate:note_description', {
+                            timestamp: translatorNote?.timestamp
+                              ? new Date(
+                                  translatorNote?.timestamp
+                                ).toLocaleString()
+                              : '',
+                            authorName: translatorNote?.authorName ?? '',
+                          })}
+                        </span>
+                      )}
+                      {canEditNote ? (
+                        <RichTextInput
+                          key={`translatorNote--${word.id}`}
+                          name="translatorNoteContent"
+                          value={translatorNoteContent}
+                          onBlur={async (e) => {
+                            saveTranslatorNote(e.target.value);
+                            saveTranslatorNote.flush();
+                          }}
+                          onChange={async (e) => {
+                            saveTranslatorNote(e.target.value);
+                          }}
+                        />
+                      ) : (
+                        <RichText content={translatorNoteContent} />
+                      )}
+                    </div>
+                    <div className="flex flex-col gap-2">
+                      <h2 className="font-bold">{t('translate:footnotes')}</h2>
+                      {footnote?.authorName && (
+                        <span className="italic">
+                          {t('translate:note_description', {
```

Comment:
Everyone can view footnotes, so we will want to conditionally render these differently. We don't need to check that they are a language member before showing them the note. That's the difference between these and translator's notes

Reply:
Ah, makes sense 👍

Reply:
Done. Also: now, only the language members and platform admin can see the note description.